index.ts

```diff
@@ -76,6 +77,20 @@ const FEATURE_OPTIONS = [
     value: 'prettier',
     label: language.needsPrettier.message,
   },
+  {
+    value: 'experimental-features',
+    label: language.needsExperimental.message,
```

Comment:
I prefer not to make "Enable experimental features" a feature selection.
Just looking at this description will not provide users with enough knowledge about the possible experimental features to make an informed decision.

So, let's just ask all the users about the experimental features. As long as the default is none, it won't disrupt the workflow.

Reply:
Does oxlint keep the original logic?

Reply:
Sorry for failing to express it clearly in the first place 😂 Had a hard time to come up with the correct words.